clover_blocks/www/blockly/blockly_compressed.js

```diff
@@ -1406,4 +1406,4 @@ return Blockly;
 }));
 
 
-//# sourceMappingURL=blockly_compressed.js.map
+//# sourceMappingURL=blockly_compressed.js.map
```

Comment:
Trailing newline in source code files is needed, so incorrect change. Please, arrange your editor so it would not remove required trailing newlines and don't commit this.